src/utils.ts

```diff
@@ -53,25 +57,27 @@ const hasConflictBetween = (section1, section2) =>
     )
   );
 
-const classes = (...classList) => classList.filter((c) => c).join(' ');
+const classes = (...classList: (string | boolean | null | undefined)[]) =>
+  classList.filter((c) => c).join(' ');
 
-const isMobile = () => window.innerWidth < 1024;
+const isMobile = (): boolean => window.innerWidth < 1024;
 
-const simplifyName = (name) => {
+const simplifyName = (name: string): string => {
   const tokens = name.split(' ');
   const firstName = tokens.shift();
   const lastName = tokens.pop();
   return [firstName, lastName].join(' ');
 };
 
-const unique = (array) => [...new Set(array)];
+const unique = <T>(array: T[]): T[] => Array.from(new Set(array));
```

Comment:
TypeScript doesn't like spreading the Set into an array on our target ES script version (es5), so we use `Array.from` instead.